projects/aca-content/src/lib/components/info-drawer/metadata-tab/metadata-tab.component.spec.ts

```diff
@@ -251,26 +252,70 @@ describe('MetadataTabComponent', () => {
     });
   });
 
-  describe('displayAspect', () => {
+  describe('ContentMetadataCardComponent', () => {
+    const getContentMetadataCard = (): ContentMetadataCardComponent =>
+      fixture.debugElement.query(By.directive(ContentMetadataCardComponent)).componentInstance;
+
     beforeEach(() => {
       fixture = TestBed.createComponent(MetadataTabComponent);
-      store = TestBed.inject(Store);
       component = fixture.componentInstance;
     });
 
-    it('show pass empty when store is in initial state', () => {
-      const initialState = fixture.debugElement.query(By.css('adf-content-metadata-card'));
-      expect(initialState.componentInstance.displayAspect).toBeFalsy();
+    describe('displayAspect', () => {
+      beforeEach(() => {
+        store = TestBed.inject(Store);
+      });
+
+      it('show pass empty when store is in initial state', () => {
```

Comment:
I think description should start from `should`

Reply:
done